encoding/wkt/wkt_encode.go

```diff
@@ -4,65 +4,91 @@ import (
 	"errors"
 	"fmt"
 	"io"
-	"reflect"
 	"strconv"
 
 	"github.com/go-spatial/geom"
+	"github.com/go-spatial/geom/cmp"
 )
 
-func isNil(a interface{}) bool {
-	defer func() { recover() }()
-	return a == nil || reflect.ValueOf(a).IsNil()
-}
-
 type Encoder struct {
 	w io.Writer
 	fbuf []byte
+
+	// Strict causes the encoder to return errors if the geometries have empty
+	// sub geometries where not allowed by the wkt spec. When Strict
```

Comment:
`Strict causes the encoder to return error if the geometries have empty sub geometries where not allowed by the wkt spec`
This hard to understand upon first read through. I'm not sure if there is a better way to state this. If there isn't this is fine –– I'm having a hard time coming up with a better way to state it.